3.0.0 release (#1402)
snowflakedb/snowflake-connector-python: DESCRIPTION.md

# Release Notes

- v2.9.1(January 23, 2023)

>   - Fixed a bug where write_pandas did not use the schema and database provided to it

Reviewer: ```suggestion
  - Fixed a bug where write_pandas did not use the schema and database provided to it when creating intermediate objects (Credit @jekaterinakletnaja)
```

Author: Sorry, calling out this contributor by handle wouldn't be fair to all of the previous contributors.
```suggestion
  - Fixed a bug where write_pandas did not use user-specified schema and database to create intermediate objects
```

  - Fixed a bug where HTTP response code of 429 were not retried
  - Fixed a bug where MFA token caching was not working
  - Bumped pyarrow dependency from >=8.0.0,<8.1.0 to >=10.0.1,<10.1.0
  - Bumped pyOpenSSL dependency from <23.0.0 to <24.0.0

>   - We now print the SSO url before attempting to open it, just in case opening the link fails

Reviewer: ```suggestion
  - In browser-based authentication, we now print the SSO url before attempting to open it in the browser, just in case opening the link fails
```

>   - Updated the level of a log for when ArrowResult cannot be imported

Reviewer: ```suggestion
  - Increased the level of a log for when ArrowResult cannot be imported
```
